internal/server/server.go

```diff
@@ -49,7 +49,24 @@ func RunServer(config *Config) error {
 	logger := log.NewCtxLogger(config.LogLevel, []string{domain.TraceIDKey})
 	crypto := crypto.NewAES(config.EncryptionSecretKeyKey)
 	validator := validator.New()
-	notifier, err := notifiers.NewClient(&config.Notifier, logger)
+	notifierConfig := notifiers.ConfigMultiClient{}
+	if &config.Notifier != nil {
+		// map old to the new format
+		var provider = config.Notifier.Provider
+		notifierConfig = notifiers.ConfigMultiClient{
+			Notifiers: map[string]notifiers.Notifier{
+				provider: {
+					Provider:    provider,
+					AccessToken: config.Notifier.AccessToken,
+					Criteria:    ".send_to_slack == true",
+				},
+			},
+		}
+	} else {
+		notifierConfig = config.Notifiers
+
+	}
```

Comment:
let's prioritize `config.Notifiers` if it presents and fallback to config.Notifier